setup.py

```diff
@@ -11,12 +11,14 @@
 extras_require["plot"] = [
     "matplotlib >=3.0",
     "scipy >=1.4",
-    "uncertainties >=3",
+    "iminuit >=2",
     "mplhep >=0.2.16",
 ]
 
 extras_require["test"] = [
+    *extras_require["plot"],
     "pytest >=4.6",
+    "pytest-mpl >=0.12",
```

Comment:
I actually thought we were already using pytest-mpl. Guess I've been staring at mplhep too long. Thanks for adding it!